banzai_floyds/tests/test_e2e.py

```diff
@@ -107,6 +113,25 @@ def test_that_order_mask_exists(self):
             # Note there are only two orders in floyds
             assert np.max(hdu['ORDERS'].data) == 2
 
```

Comment:
Our original e2e tests only included logic that the files were created but nothing about the actual reduction quality. Here we add tests that we identified the centers of the orders correctly and that our wavelength solution matches a a solution I made by hand in the manual reduction jupyter notebook.